tests/modules/services/window-managers/sway/sway-followmouse-legacy.nix

```diff
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    wayland.windowManager.sway = {
+      enable = true;
+
+      config = {
+        focus.followMouse = false;
+        menu = "${pkgs.dmenu}/bin/dmenu_run";
+        terminal = "${pkgs.termite}/bin/termite";
+        bars = [{ command = "${pkgs.swaybar}/bin/swaybar"; }];
+      };
+    };
+
+    nixpkgs.overlays = [
+      (self: super: {
+        dmenu = super.dmenu // { outPath = "@dmenu@"; };
+        rxvt-unicode = super.rxvt-unicode // { outPath = "@rxvt-unicode@"; };
+        sway = super.sway // { outPath = "@sway@"; };
+        swaybar = super.sway-unwrapped // { outPath = "@swaybar@"; };
+        termite = super.sway-unwrapped // { outPath = "@termite@"; };
```

Comment:
Should this really be `super.sway-unwrapped`?

Reply:
No. I'll fix this. Thanks!

Reply:
(It also doesn't really matter though, since the important part is faking out the store path to "@termite@" to give us a static string to test against)